doc/content.tex

```diff
@@ -1,1347 +0,0 @@
-    test
-    is a negative ADF statistics. The more negative it is the stronger the rejection of the hypothesis.
-    The full form of ADF test is listen in \ref{eq:adf-test}.
-
-    \begin{gather} \label{eq:adf-test}
-        \Delta y_t = \alpha + \beta t + \gamma \Delta y_{t-1} + \dots + \delta{p-1} \Delta y_{t-p+1} + \epsilon_t \\
-        ADF = \frac{\hat{\gamma}}{SE(\hat{\gamma})} \label{eq:adf-stat}
-    \end{gather}
-
-    There are multiple variants of ADF test. Some of them leave out parts of equation \ref{eq:adf-test}. The most
-    important ones and widely used are:
-
-    \begin{itemize}
-        \item \emph{nc}\,--\,for regression with no constant nor time trend ($\beta t$)
-        \item \emph{c}\,--\,for regression with an intercept but no time trend ($\beta t$)
-        \item \emph{ct}\,--\,for regression with an intercept and time trend
-    \end{itemize}
-
-    Each of them is good for testing particular type of stationarity. For example for testing if there is trend
-    present in the time series \emph{c} version is the best choice.
-
-    The implementation of this test fits multiple linear regression model from equation \ref{eq:adf-test} and
-    calculates ADF statistics listed in \ref{eq:adf-stat}. SE denotes standard error of estimated $\hat{\gamma}$.
-
-    %%%%%%%%
-    \section{Seasonality Detection} \label{sec:period-detection}
-    Forecasting engine in Hawkular system does not have any inside information about period of a time series being
-    modelled. Therefore automatic period identification has to be implemented. In practice it is a difficult task and
-    result often differs from correct period, specially if there is significant noise present in the series
-    \cite{period-meteo}.
-
-    There are several approaches how to implement automatic period identification. The most used ones are based on
-    autocorrelation function (ACF) or spectral density \cite{period-hydman}. This work applies ACF method.
-
-    In the following chart \ref{img:period-acf} ACF function of sine function is shown. The period of this
-    function is seven. There are patterns repeated every seven observations and it is decreasing to zero.
-
-    \begin{figure}[H]
-        \begin{center}
-            \scalebox{0.73}[0.6]{\includegraphics{img/acf-sine.pdf}}
-            \caption{Autocorrelation function of sine function.}
-            \label{img:period-acf}
-        \end{center}
-    \end{figure}
-
-    The algorithm for automatic period identification is demonstrated in \ref{alg:period-find}.
-    It is looking for periodically repeated significant values of ACF function. At some rate these values have to
-    be decreasing to zero. In the infinity ACF function converges to zero.
-
-    The algorithm as input takes only time series. If there is a unit root present differencing is applied.
-    Follows calculation of autocorrelation function of input time series and finding the index of its highest
```

Comment:
Po slovensky som to chcel povedat takto: Ak je tam pritomny jednotkovy koren aplikuje sa difrencovanie. Nasleduje vypocet autokorelacnej funkcie